org.chromium.Chromium.yaml

```diff
@@ -0,0 +1,248 @@
+app-id: org.chromium.Chromium
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: chromium
+finish-args:
+  - --require-version=1.8.2
+  - --filesystem=host-etc
```

Comment:
Why this is needed?

Reply:
If you're talking about `host-etc` it's probably because Chromium should be able to read `/etc/hosts`

Reply:
Any app can read /etc/hosts without that permission.

Reply:
I believe this is to support the policies shipped by the distro - on Endless we ship some chromium policies under `/etc/chromium-browser/policies`...

Reply:
Aren't those policies part of chromium distro package and supposed to be used with it? Chromium upstream moves very fast so you may not even be sure if distro policies are still valid for flatpak version.

Reply:
Beside that the current policy path seems to be [/etc/chromium](https://source.chromium.org/search?q=%2Fetc%2Fchromium) so I wonder how it's supposed  to work for Endless.

Reply:
I see that you ship special patch in Endless that adds support for the [other path](https://github.com/endlessm/chromium-browser/commit/320b8162520df2488fa647cdbf2aaf7eae5b7ddb). I think this patch isn't part of this submission so your policies won't work. If patching is considered ok here then I recommend choosing path outside `/etc` in order to not having to open additional sandbox hole in flatpak and change it accordingly for Endless.

Reply:
@Erick555, that patch is originated from Debian because the package was named chromium-browser, because chromium-bsu pre-dated it in the archive, and it took 10 years to switch them or something. :)

Normally /etc is admin controlled, rather than being the best place for distro policies (although on Endless a lot of the stuff in /etc ends up being unmodified from what the ostree ships) but IMO its not unusual to expect a Flatpak to also respect the administrator policies.

We use it to manage some child-safety stuff, DNS, safesearch, etc if its enabled by the administrator. Endless could change to use /etc/chromium (the upstream default path) at the same time as migrating to the Flatpak if that would be desirable.

Does flatpak not allow --filesystem=/etc/chromium or so?

Reply:
> Does flatpak not allow --filesystem=/etc/chromium or so?

It doesn't which is the problem here as you have to leak entire `/etc` in sandbox.

Snap moved to `$SNAP_DATA/policies` (which resolves to something like `/var/snap/chromium/common` and I think finding similar flatpak friendly path that would work with `--filesystem` should desired here. It could be somewhre under `/var` too.

Reply:
In this case, we may as well provide a merged extension point for this folder. The distro can provide it as an unmanaged extension in /var/lib/flatpak/extension/... or the user can put whatever there if they prefer, or even download Flatpaks to do stuff (add Chromium native extensions?). :)